psi4/driver/driver_cbs.py

```diff
@@ -1122,25 +1635,37 @@ def cbs(func, label, **kwargs):
 
        - Need to add more extrapolation schemes
 
-    As represented in the equation below, a CBS energy method is defined in several
-    sequential stages (scf, corl, delta1, delta2, ... ) covering treatment
-    of the reference total energy, the correlation energy, a delta correction to the
-    correlation energy, and a second delta correction, etc.. Each is activated by its
-    stage_wfn keyword, or as a field in the ```cbs_metadata``` list, and is only
-    allowed if all preceding stages are active.
+    As represented in the equation below, a CBS energy method is defined in 
+    several sequential stages (scf, corl, delta1, delta2, ... ) covering treatment
+    of the reference total energy, the correlation energy, a delta correction to 
+    the correlation energy, and a second delta correction, etc. Each is activated 
+    by its stage_wfn keyword, or as a field in the ```cbs_metadata``` list, and 
+    is only allowed if all preceding stages are active.
+
+    The extrapolation of DFT calculations from basis sets of successive zeta 
+    levels is also implemented. By default, the components of the DFA recipe
+    (fctl, dh, disp, nl) are treated separately: the fctl component (equivalent
+    to the scf stage in WFT) is extrapolated using the exponential -- square root
+    function; the dh component (equivalent to the corl stage in WFT) is
+    extrapolated using the power function; and the disp and nl components are not
+    extrapolated at all.
 
     .. include:: /cbs_eqn.rst
 
-    * Energy Methods
-        The presence of a stage_wfn keyword is the indicator to incorporate
-        (and check for stage_basis and stage_scheme keywords) and compute
-        that stage in defining the CBS energy.
+    .. _cbs-energy-methods:
+
+    Energy Methods
+    --------------
+    
+    The presence of a stage_wfn keyword is the indicator to incorporate (and check
+    for stage_basis and stage_scheme keywords) and compute that stage in defining 
+    the CBS energy.
 
-        The cbs() function requires, at a minimum, ``name='scf'`` and ``scf_basis``
-        keywords to be specified for reference-step only jobs and ``name`` and
-        ``corl_basis`` keywords for correlated jobs.
+    The cbs() function requires, at a minimum, ``name='scf'`` and ``scf_basis``
+    keywords to be specified for reference-stage only jobs and ``name`` and
+    ``corl_basis`` keywords for correlated jobs.
```

Comment:
Does correlated job mean, "job that extrapolates the correlation energy" or "job that only extrapolates the correlation energy"?

Reply:
Yeah, this is a good question. It should be only the correlated energy, but I have a feeling it's the total energy. I'll pass this question to the higher ups @loriab.

Reply:
It means the former. But if you only specify the name and corl_basis, then the reference stage is going to get the highest_1 treatment, so it also means the latter.

Reply:
Okay, so it means "job that extrapolates the correlation energy" as one component, agnostic to whether other components are extrapolated. Peter, can you update the docstring accordingly?